tests/test_table.py

```diff
@@ -648,6 +648,32 @@ def test_exceptions():
         )
         db['table']['column'] = audformat.Column(rater_id='invalid')
 
+    # level and column names must be unique
+
```

Comment:
Could you add matches for the error message here, so we make sure not a different `ValueError` is raised

Reply:
I tried with:

```
            marks=pytest.mark.xfail(
                raises=ValueError,
                match='index level with same name',
            ),
```

but it does seem to have an effect as I can put anything with `match=...` and it still works.

Reply:
No, not with `marks`, I'm talking about the tests after this line that all start with `with pytest.raises(ValueError)`, there you can add `with pytest.raises(ValueError, match=error_msg)`